src/worker_http.rs

```diff
@@ -99,7 +97,14 @@ async fn image_payload_from_url(
 
             // Encode the image if needed
             let mut image_bytes: Vec<u8> = Vec::new();
+
             if encode_images {
+                // If the image is 16 bits or 2 channels, standardize to 8 bits, 3 channels
```

Comment:
fix an issue @tarek-ayed had, I'm adding a unit test to nail this, if re-encoding was requested then we normalize to rgb8. Could be a setting we expose down the line